integration/nwo/fsc/fsc.go

```diff
@@ -553,16 +552,11 @@ func (p *Platform) GenerateCoreConfig(peer *node2.Replica) {
 }
 
 func (p *Platform) PersistenceOpts(prefix string, uniqueName string, o *node2.Options, dirName string) node2.PersistenceOpts {
-	if sqlOpts := o.GetPersistence(prefix); sqlOpts != nil {
-		return node2.PersistenceOpts{
-			Type: sql.SQLPersistence,
-			SQL:  sqlOpts,
-		}
-	} else {
-		return node2.PersistenceOpts{
-			Type:   badger.BadgerPersistence,
-			Badger: &node2.BadgerOpts{Path: p.NodeStorageDir(uniqueName, dirName)},
-		}
+	sqlOpts := o.GetPersistence(prefix)
```

Comment:
This method can now be static. We can remove the other one because it's a duplicate